packages/bridge-ui/src/utils/selectToken.ts

```diff
@@ -0,0 +1,27 @@
+import { get } from 'svelte/store';
+
+import { BridgeType } from '../domain/bridge';
+import type { Token } from '../domain/token';
+import { bridgeType } from '../store/bridge';
+import { token } from '../store/token';
+import { isETH } from '../token/tokens';
+import { getLogger } from '../utils/logger';
+
+const log = getLogger('util:selectToken');
+
+export function selectToken(_token: Token) {
```

Comment:
`selectTokenAndBridgeType` ?
this function has more consequences than selecting a token

Reply:
I thought about it, but it's kind of implied here, no?: the type of asset is tight to the bridge used... but thinking about it, it's a good point, it changes state that might be important to mention here, and without comments, someone debugging (I'm thinking about stack trace here) could miss a bug... I'll change that in the next PR

Reply:
Done